### PR TITLE
feat: --ignore-fragments CLI parameter

### PR DESCRIPTION
**Pull request by zregvart**
This adds `--ignore-fragments` parameter to the command line, when
specified no fragment checking will be performed.

@jyn514, this is my first quick pass at this, happy to incorporate any suggestions you might have.

### CODE REVIEW

File: src/bin/cargo-deadlinks.rs [hunk 1]
```diff
@@ -21,6 +21,7 @@ Options:
     -h --help               Print this message.
     --dir                   Specify a directory to check (default is target/doc/<package>).
     --check-http            Check 'http' and 'https' scheme links.
+    --ignore-fragments      Don't check fragments
```
Comment (jyn514) [inline src/bin/cargo-deadlinks.rs:24]:
```suggestion
    --ignore-fragments      Don't check URL fragments.
```

File: src/bin/deadlinks.rs [hunk 2]
```diff
@@ -16,6 +16,7 @@ Usage:
 Options:
     -h --help               Print this message
     --check-http            Check 'http' and 'https' scheme links
+    --ignore-fragments      Don't check fragments
```
Comment (jyn514) [inline src/bin/deadlinks.rs:19]:
```suggestion
    --ignore-fragments      Don't check URL fragments.
```

**Review by jyn514 — COMMENTED** [review]
Looks great, thanks! Can you add a quick line to the changelog?

**Comment by zregvart** [comment]
@jyn514 thanks, done :)

**Pull request merged**